src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowDiversityV2Spec.groovy

```diff
@@ -73,8 +76,27 @@ class FlowDiversityV2Spec extends HealthCheckSpecification {
         }
         allInvolvedIsls.unique(false) == allInvolvedIsls
 
-        cleanup: "Delete flows"
+        and: "Flows' histories contain 'groupId' information"
+        [flow2, flow3].each {//flow1 had no diversity at the time of creation
```

Comment:
in case flow has no diversity, then groupId is missed
am I right?

Reply:
It is not missed, it just does not exist